src/Umbraco.Cms.Integrations.Automation.Zapier/Services/UserValidationService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+
+#if NETCOREAPP
+using Umbraco.Cms.Core.Security;
+using Umbraco.Cms.Core.Services;
+#else
+using Umbraco.Core.Services;
+using Umbraco.Web;
+#endif
+
+namespace Umbraco.Cms.Integrations.Automation.Zapier.Services
+{
+    public class UserValidationService : IUserValidationService
```

Comment:
Nice, I like how you've wrapped this up to share to different controllers and to keep the V8/V9 logic in one place.

Reply:
Thanks.